jetty-core/jetty-siwe/pom.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.eclipse.jetty</groupId>
+    <artifactId>jetty-core</artifactId>
+    <version>12.0.11-SNAPSHOT</version>
+  </parent>
+  <artifactId>jetty-siwe</artifactId>
+  <name>Core :: Sign-In with Ethereum</name>
+  <description>Jetty Sign-In with Ethereum</description>
+
+  <properties>
+    <bundle-symbolic-name>${project.groupId}.siwe</bundle-symbolic-name>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.jetbrains.kotlin</groupId>
+        <artifactId>kotlin-stdlib-common</artifactId>
+        <version>1.9.10</version>
+      </dependency>
+      <dependency>
+        <groupId>org.jetbrains.kotlin</groupId>
+        <artifactId>kotlin-stdlib-jdk7</artifactId>
```

Comment:
A) Why we need to depend on Kotlin, and B) why the JDK 7 version?

Reply:
This is brought in through the web3j dependency.

Turns out it compiles fine with only the jdk8 version in the dependency management.
But it does have `org.jetbrains.kotlin:kotlin-stdlib-jdk7:jar:1.8.21:compile` in the dependency tree.

This is the error 
```
[ERROR] Rule 4: org.apache.maven.enforcer.rules.dependency.RequireUpperBoundDeps failed with message:
[ERROR] Failed while enforcing RequireUpperBoundDeps. The error(s) are [
[ERROR] Require upper bound dependencies error for org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.8.21 paths to dependency are:
[ERROR] +-org.eclipse.jetty:jetty-siwe:12.0.11-SNAPSHOT
[ERROR]   +-org.web3j:core:4.12.0
[ERROR]     +-com.squareup.okhttp3:okhttp:4.12.0
[ERROR]       +-org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.8.21
[ERROR] and
[ERROR] +-org.eclipse.jetty:jetty-siwe:12.0.11-SNAPSHOT
[ERROR]   +-org.web3j:core:4.12.0
[ERROR]     +-com.squareup.okhttp3:logging-interceptor:4.12.0
[ERROR]       +-org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.8.21
[ERROR] and
[ERROR] +-org.eclipse.jetty:jetty-siwe:12.0.11-SNAPSHOT
[ERROR]   +-org.web3j:core:4.12.0
[ERROR]     +-io.tmio:tuweni-bytes:2.4.2
[ERROR]       +-org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.8.21
[ERROR] and
[ERROR] +-org.eclipse.jetty:jetty-siwe:12.0.11-SNAPSHOT
[ERROR]   +-org.web3j:core:4.12.0
[ERROR]     +-io.tmio:tuweni-units:2.4.2
[ERROR]       +-org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.8.21
[ERROR] and
[ERROR] +-org.eclipse.jetty:jetty-siwe:12.0.11-SNAPSHOT
[ERROR]   +-org.web3j:core:4.12.0
[ERROR]     +-com.squareup.okhttp3:okhttp:4.12.0
[ERROR]       +-com.squareup.okio:okio:3.6.0
[ERROR]         +-com.squareup.okio:okio-jvm:3.6.0
[ERROR]           +-org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.9.10
[ERROR] ]

```